### PR TITLE
refactor: rebranding Intelligent Adaptive Access to Risk-based MFA

### PR DESCRIPTION
**Pull request by remisultan**
## :id: Reference related issue. 

https://github.com/gravitee-io/issues/issues/7552

## :pencil2: A description of the changes proposed in the pull request

This PR rebrands Intelligent Adaptive Access to Risk-based MFA.
It also removes the consent checkboxes from the default template and moves them to the template examples in the form design

### CODE REVIEW

File: gravitee-am-ui/src/app/domain/components/forms/form/dialog/form-info.component.html [hunk 1]
```diff
@@ -35,6 +35,21 @@ <h1 matDialogTitle>Write your own {{data.template}} page</h1>
...
+                </span>
+            </label>
+            <br>
+            <label>
+                <input type="checkbox" th:checked="${uc_ua}" id="uc_ua" name="uc_ua">
+                <span id="#label-consent-ua">I consent to the collection and use of User-Agent information for audit log purposes
+                    <span th:if="${userActivityEnabled}" th:text="${'and to improve security for a duration of ' + user_activity_retention_time}"></span>
+                </span>
+            </label>
+            <br>
```
Comment (leleueri) [inline gravitee-am-ui/src/app/domain/components/forms/form/dialog/form-info.component.html:52]: Maybe we should display a text message before the HTML snippet to explain that these two field are optional and useful only for RiskAssessment / Geo IP.

**Review by leleueri — APPROVED** [review]
I approve because there are no issues reagrding the code.
But I put two comments regarding the HTML snippet. An explanation is missing IMHO but if PO's validate this version it is OK for me otherwise please discuss with them :) .

File: gravitee-am-ui/src/app/domain/components/forms/form/dialog/form-info.component.html (hunk 1 above)
Comment (remisultan) [reply gravitee-am-ui/src/app/domain/components/forms/form/dialog/form-info.component.html:52]: FP

File: gravitee-am-ui/src/app/domain/components/forms/form/dialog/form-info.component.html [hunk 2]
```diff
@@ -190,7 +222,7 @@ <h1 matDialogTitle>Write your own {{data.template}} page</h1>
     </pre>
     <p *ngIf="data.rawTemplate === 'FORGOT_PASSWORD'">
       <b>NOTE:</b> When forgot password succeeded, the token present into the URL is used to populate user profile into the template engine. If the token is invalid or has expired, the user profile will be missing from the template engine context, so pay attention to test if user is null before accessing its attributes.
-      Here after, you can find a snippet of HTML to adapt the message to inform the user on whoch address email the message has been sent. 
+      Here after, you can find a snippet of HTML to adapt the message to inform the user on whoch address email the message has been sent.
```
Comment (Johngren) [inline gravitee-am-ui/src/app/domain/components/forms/form/dialog/form-info.component.html:225]: Which*

**Review by leleueri — APPROVED** [review]
